app-text/cutemarked/cutemarked-0.11.3.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Qt Markdown editor"
+DESCRIPTION="A Qt-based, free and open source Markdown editor with live HTML \
+preview, math expressions, code and markdown syntax highlighting."
+HOMEPAGE="https://cloose.github.io/CuteMarkEd"
+COPYRIGHT="2013 Christian Loose
+	2007 David Loren Parsons
+	2011-2012 Ali Rantakari
+	2012 CodeCatalyst, LLC"
+LICENSE="GNU GPL v2
+	MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/cloose/CuteMarkEd/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="78a41808c2f0452375810abdff76eeaaee012f8d1368a2b8772ec6b4d2ceeec8"
+SOURCE_DIR="CuteMarkEd-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cutemarked$secondaryArchSuffix = $portVersion
+	app:CuteMarkEd$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libhunspell_1.6$secondaryArchSuffix
```

Comment:
```diff
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
 	lib:libhunspell_1.6$secondaryArchSuffix
```
BTW, I found that libGL was missing in REQUIRES by using haikuporter's  `-S` option.